angular/src/components/icon.component.ts

```diff
@@ -104,4 +106,16 @@ export class IconComponent implements OnChanges {
             this.image = null;
         }
     }
+
+
+    private setCardIcon() {
+        if (this.imageEnabled) {
+            try {
+                this.image = 'images/cards/' + this.cipher.card.brand + '.png';
+                this.fallbackImage = 'images/cards/Other.png';
```

Comment:
My thought was that you might at some point add a brand the user can choose, but perhaps not add an icon for it or forget to add to one of the clients. This would ensure that a card icon is shown instead of an empty box in that scenario.

Maybe it is not something you expect to happen? I can remove the fallback in that case.

Reply:
Valid point, but in this scenario I think it makes more sense to have a `Record` of valid card brands and their respective icon.

Something like:
```typescript
cardIcons: Record<string, string> = {
'American Express': 'american_express',
....
]

if (this.cipher.card.brand in cardIcons) {
    this.image = 'images/cards/' + cardIcons[this.cipher.card.brand] + '.png';
}
```

It should be a bit more future safe since brands could be added safely without having the clients attempt to load arbitrary images. And it also removes the space and capital letters from the filenames.

Reply:
Great point, I like this solution! Will add it together with the rest of the small fixes in the coming days.
Thanks for all the input :)